shared/locales/en/feedback.json

```diff
@@ -0,0 +1,54 @@
+{
+    "smartling": {
+        "string_format" : "icu",
+        "translate_paths": [{
+            "path": "*/title",
+            "key": "{*}/title",
+            "instruction": "*/note"
+        }]
+    },
+    "brokenSiteLabel": {
+        "title": "Which website is broken?",
+        "note": "Input label for broken site URL"
+    },
+    "brokenSitePlaceholder": {
+        "title": "Copy and paste your URL",
+        "note": "Placeholder text on URL input"
+    },
+    "feedbackHeaderLabel": {
+        "title": "What do you love? What isn't working? How could the extension be improved?",
+        "note": "Label over feedback textarea"
+    },
+    "feedbackPlaceholder": {
+        "title": "Which features or functionality does your feedback refer to? Please be as specific as possible.",
+        "note": "Placeholder text on feedback form input."
+    },
+    "submittingFeedbackHelps": {
+        "title": "Submitting anonymous feedback helps us improve DuckDuckGo Privacy Essentials.",
+        "note": "Description at top of feedback form."
+    },
+    "reportBrokenSite": {
+        "title": "I want to report a broken site",
+        "note": "Label for broken site report toggle"
```

Comment:
"This opens a feedback form to report a broken site. Broken here means 'not working as expected'."